src/context/mod.rs

```diff
@@ -229,6 +229,13 @@ pub trait Context<'a>: ContextInternal<'a> {
         result
     }
 
+    #[cfg(feature = "try-catch-api")]
+    fn try_catch<'b: 'a, F>(&mut self, f: F) -> Result<Handle<'a, JsValue>, Handle<'a, JsValue>>
+        where F: UnwindSafe + FnOnce(&mut Self) -> JsResult<'b, JsValue>
```

Comment:
For the final non-feature flagged version, I think we should have separate legacy and N-API function signatures so that we don't need to unnecessarily require `UnwindSafe` on N-API. Removing the trait bound would be a breaking change.

Reply:
Good point! It's maybe not even a breaking change, since anything that is `UnwindSafe` will still satisfy the relaxed restriction.